plugins/node/opentelemetry-instrumentation-aws-sdk/src/services/bedrock-runtime.ts

```diff
@@ -23,19 +32,57 @@ import {
   ATTR_GEN_AI_REQUEST_TEMPERATURE,
   ATTR_GEN_AI_REQUEST_TOP_P,
   ATTR_GEN_AI_REQUEST_STOP_SEQUENCES,
+  ATTR_GEN_AI_TOKEN_TYPE,
   ATTR_GEN_AI_USAGE_INPUT_TOKENS,
   ATTR_GEN_AI_USAGE_OUTPUT_TOKENS,
   ATTR_GEN_AI_RESPONSE_FINISH_REASONS,
   GEN_AI_OPERATION_NAME_VALUE_CHAT,
   GEN_AI_SYSTEM_VALUE_AWS_BEDROCK,
+  GEN_AI_TOKEN_TYPE_VALUE_INPUT,
+  GEN_AI_TOKEN_TYPE_VALUE_COMPLETION,
 } from '../semconv';
 import {
   AwsSdkInstrumentationConfig,
   NormalizedRequest,
   NormalizedResponse,
 } from '../types';
+import {
+  hrTime,
+  hrTimeDuration,
+  hrTimeToMilliseconds,
+} from '@opentelemetry/core';
 
 export class BedrockRuntimeServiceExtension implements ServiceExtension {
+  private tokenUsage!: Histogram;
+  private operationDuration!: Histogram;
+
+  updateMetricInstruments(meter: Meter) {
+    this.tokenUsage = meter.createHistogram('gen_ai.client.token.usage', {
```

Comment:
nit: A comment linking to https://opentelemetry.io/docs/specs/semconv/gen-ai/gen-ai-metrics/#metric-gen_aiclienttokenusage could be helpful. Ditto for the other metric below.